packages/core/src/utils/composeAutoAccept.ts

```diff
@@ -0,0 +1,14 @@
+import { AutoAcceptCredential } from '../modules/credentials/CredentialAutoAcceptType'
+/**
+ * Returns the credential auto accept config based on priority:
+ *	- The record config takes first priority
+ *	- Otherwise the agent config
+ *	- Otherwise {@link AutoAcceptCredential.Never} is returned
+ */
+
+export function composeAutoAccept(
```

Comment:
As this is a credential specific util (it uses `AutoAcceptCredential` enum type) can we move this to the credentials module directory?